examples/tutorials/data/hawc.py

```diff
@@ -1,7 +1,12 @@
 """
 HAWC with Gammapy
-=====================
-Explore HAWC event lists and IRFs and perform the data reduction steps.
+=================
+
+Explore HAWC event lists and instrument response functions (IRFs), then perform the
+data reduction steps.
+
+Introduction
```

Comment:
Note that you might add the `Introduction ` section in the HESS tutorial as well as it lacks it.